package.json

```diff
@@ -52,21 +56,27 @@
     "pinia": "2.0.29",
     "proxy-from-env": "^1.1.0",
     "vee-validate": "^3.4.14",
+    "vue": "2.6.14",
```

Comment:
Why would you add it?

Reply:
We've 3 references in this storefront code to `vue` package. Eslint reports `import/no-extraneous-dependencies` errors.

Should I disable this rule for those 3 occurrences and remove this dependency?

Reply:
Ok, if we have direct references it must be added.